ts/src/binance.ts

```diff
@@ -1335,103 +1335,9 @@ export default class binance extends Exchange {
                     'SPL': 'SOL', // temporarily keep support for SPL (old name)
                     'SOL': 'SOL', // we shouldn't rename SOL
                 },
-                // keeping this object for backward-compatibility
```

Comment:
this comment was maybe 4 years ago, and either, where this "old" object was referenced, that code of part has not been used in binance till date.
so, this place just adds confusion to binance class and readability. we don't really need this